nixos/modules/services/networking/nebula.nix

```diff
@@ -88,6 +88,24 @@ in
               example = [ "192.168.100.1" ];
             };
 
+            dns.enable = mkOption {
+              type = types.bool;
+              default = false;
+              description = lib.mdDoc "Whether this node should serve DNS.";
+            };
+
+            dns.host = mkOption {
+              type = types.str;
+              default = "[::]";
+              description = lib.mdDoc "the interface to serve DNS on.";
```

Comment:
This probably should just be something like `localhost`? And maybe change the doc to "The address to serve DNS on."

Reply:
Nebula does recommend using that value to bind to all interfaces: https://nebula.defined.net/docs/guides/using-lighthouse-dns/#configuration

Will update the description